src/music/tests/tests_models.py

```diff
@@ -0,0 +1,16 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+
+from core.utils.samples import sample_track
+
+
+class TestMusicModel(TestCase):
+    def setUp(self):
+        self.test_track = sample_track(title="test_track")
+
+    def tearDown(self):
+        self.test_track.delete()
+
+    def test_title_limit(self):
```

Comment:
It's a good practice to name your test functions with a descriptive name, such as test_track_title_limit or test_track_title_max_length.